src/Sarif.WorkItemFiling/AzureDevOpsClientWrapper.cs

```diff
@@ -123,7 +119,7 @@ public override async Task<IEnumerable<WorkItemModel>> FileWorkItems(IEnumerable
                 try
                 {
                     Console.Write($"Creating work item: {metadata.Title}");
-                    workItem = await _witClient.CreateWorkItemAsync(patchDocument, project: _projectName, "Bug");
+                    workItem = await _witClient.CreateWorkItemAsync(patchDocument, project: this.ProjectOrRepository, "Bug");
```

Comment:
>Bug [](start = 119, length = 3)

Should we change this default now or later? Currently AzureDevOps defaults work item type as "Issue".

Reply:
I like using the default for now, though this should be one of our first customization points.

Reply:
An issue is a task in the ADO parlance. I think tasks are likely not to be the default for an automated filing system (i.e., bugs would be). Interested in others thoughts on this. Agree with Nick, let's defer a change for now.

---
In reply to: [376555448](https://github.com/microsoft/sarif-sdk/pull/1746#discussion_r376555448) [](ancestors = 376555448)